src/sw-fix.js

```diff
@@ -0,0 +1,90 @@
+// this gets appended to the built sw
+
+/*global clients*/
+
+// eslint-disable-next-line
+var storage = (function(e) {
+	'use strict';
```

Comment:
Travis is unhappy with this

Reply:
hmm - this was previously all on one line, which meant the ESLint ignore on L5 applied to the whole thing.